### PR TITLE
Added setup instructions

### LINKED ISSUE #62: [DOC] - Add Set Up Instructions
# Incorrect Documentation

## Describe the error

The CONTRIBUTING.md file is missing the specific set up instructions for this project.

## Expected information

In the `Setting Up Your Code` section, we should add:

- Installing dependencies (`npm ci`)
- Building files (`npm run build`)
- Setting up Discord application
- Copying `sample.env` to `.env`
- Adding values to `.env`.
- Verifying lint (`npm run lint`)

### PR DESCRIPTION
**Pull request by adityaraute**
# Pull Request

## Description:

 
Instructs the user to help set up their application.

## Related Issue:

Closes #62

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ Please ensure all pull requests and contributions comply with the [Developer Cer
 
 First, fork this repository to your own account. Then use `git clone <url>` to bring your forked repository down to your local machine (remember to get the URL for *your* repository, not the original). Optionally, use `git remote add upstream <url>` to add the original repository as the upstream (this is helpful for keeping your fork up-to-date).
 
+For running the project on your machine:
+- Install dependencies using `npm ci`
+- Execute the build command using `npm run build`
+- Set up your Discord Application. Follow this [guide](https://discordjs.guide/preparations/setting-up-a-bot-application.html) if you need help.
+- Copy the contents of the file `sample.env` to `.env`
+- Add the retrieved values to `.env`
```
Comment (nhcarrigan) [inline CONTRIBUTING.md:14]: We should detail the `env` values needed for the project.

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Rather than writing a list, let's make this some proper text. [Here's an outdated example](https://github.com/nhcarrigan-archive/Rosalia-Nightsong/blob/main/CONTRIBUTING.md).

File: CONTRIBUTING.md (hunk 1 above)
Comment (adityaraute) [reply CONTRIBUTING.md:14]: So is it okay if I just copy the values from the outdated example?

Comment (nhcarrigan) [reply CONTRIBUTING.md:14]: The values in the example are specific to that project. This project uses only one value: `TOKEN`, which is the discord bot token.

(Also noticed I don't have a sample.env so I'll get that added)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ For running the project on your machine:
 - Execute the build command using `npm run build`
 - Set up your Discord Application. Follow this [guide](https://discordjs.guide/preparations/setting-up-a-bot-application.html) if you need help.
 - Copy the contents of the file `sample.env` to `.env`
-- Add the retrieved values to `.env`
+- Add the retrieved values to `.env`. The value needed for this project will be `DISCORD_TOKEN` : The Discord page will provide you this value. Remember to put this value in quotes.
```
Comment (nhcarrigan) [inline CONTRIBUTING.md:14]:
```suggestion
- Add the retrieved values to `.env`. The value needed for this project will be `TOKEN` : The Discord page will provide you this value. Remember to put this value in quotes.

```
Just `TOKEN`. 🙂

**Review by nhcarrigan — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (nhcarrigan) [reply CONTRIBUTING.md]: Generally the contributing docs on my projects use full paragraphs to allow explanation for each step - I can go ahead and merge this as a list, and revisit later if needed though.

Comment (adityaraute) [reply CONTRIBUTING.md]: okay great then

**Review by nhcarrigan — COMMENTED** [review]
Looks good to me.

**Review by nhcarrigan — APPROVED** [review]

**Comment by nhcarrigan** [comment]
## Congratulations

Congratulations on making your first pull request to our project. We have reviewed your contributions and are happy to accept them.

We hope you enjoyed the contribution process, and encourage you to watch for issues labelled `help wanted` for additional opportunities to contribute.

**Pull request merged**